README.md

```diff
@@ -728,6 +728,30 @@ In either case:
       end
     ```
 
+* <a name="unless-with-comparator"></a>Avoid `unless` with comparators if you can use `if` with an opposing comparator.<sup>[[link](#unless-with-comparator)]</sup>
```

Comment:
We have to arbitrarily pick a standard if it's purely just about a style choice (like 2 spaces vs. 4) and we cannot find any good objective argument to support it.

Is this one of those cases? Or are we somehow trying to make the code easier to read/understand? Maybe it sounds more natural to the spoken language? There's some discussion here: https://github.com/bbatsov/ruby-style-guide/issues/329

Or maybe the rationale is it's easier to think in terms of ```if``` rather than ```unless``` because the latter is a negation.